FirebaseTestProject/Views/NewPostForm.swift

```diff
@@ -10,15 +10,19 @@ import SwiftUI
 struct NewPostForm: View {
     @State var postContent = ""
     @State var title = ""
+    @State var author = ""
+    @Environment(\.presentationMode) var presentationMode
```

Comment:
iOS15 has a new `@Environment(\.dismiss) private var dismiss` that we can use.

https://www.swiftbysundell.com/articles/dismissing-swiftui-modal-and-detail-views/

Reply:
Completed